services/jupyter/README.md

```diff
@@ -0,0 +1,21 @@
+# Jupyter Notebook
+
+This Jupyter Notebook instance is preconfigured with an example notebook that shows the usage of [Pyscicat](https://github.com/scicatproject/pyscicat).
+
+## [Pyscicat Notebook](./config/notebooks/pyscicat.ipynb)
+This notebook demonstrates all the major actions Pyscicat is capable of: 
+ * logging into SciCat backend
+ * dataset creation
+ * datablock creation
+ * attachment upload
+
+## [.env file](./config/.env)
+It contains the environment variables for connecting to the backend service deployed by this project
+
+## [Thumbnail image](./config/notebooks/example_files/thumbnail.png)
+An example image that is used for the attachment upload demonstration
+
+## Default configuration
+This service is only dependant on the backend service, since it demonstrates communication with the latter through Pyscicat.
+
+The [main readme](../../README.md) covers all dependencies of this package.
```

Comment:
can we add in this readme that personal changes to the notebooks should not be committed and users should save their changes to another file?